lib/postprocessing/image.js

```diff
@@ -93,6 +93,8 @@ const EXTRA_INSTRUCTIONS = new Set([
     "watermark"
 ]);
 
+const SRGB_COLOR_PROFILE_FILE = process.env.SRGB_COLOR_PROFILE_FILE;
```

Comment:
If a colorprofile file should be used instead of default, the color profile path should be set in this env variable (eg. as env var in Docker if a docker image gets used).